src/breadcrumbs/breadcrumbs.js

```diff
@@ -10,43 +10,35 @@ LICENSE file in the root directory of this source tree.
 import React, {Children} from 'react';
 
 import {LocaleContext} from '../locale/index.js';
-import {ChevronRight} from '../icon/index.js';
 import type {BreadcrumbsPropsT} from './types.js';
 import type {BreadcrumbLocaleT} from './locale.js';
-import {StyledRoot, StyledSeparator} from './styled-components.js';
-import {getOverrides, mergeOverrides} from '../helpers/overrides.js';
+import {StyledRoot, StyledList, StyledListItem} from './styled-components.js';
+import {getOverrides} from '../helpers/overrides.js';
 
 type LocaleT = {|locale?: BreadcrumbLocaleT|};
 export function BreadcrumbsRoot(props: {|...BreadcrumbsPropsT, ...LocaleT|}) {
   const {overrides = {}} = props;
-  const numChildren = Children.count(props.children);
   const childrenWithSeparators = [];
 
   const [Root, baseRootProps] = getOverrides(overrides.Root, StyledRoot);
-  const [Icon, baseIconProps] = getOverrides(overrides.Icon, ChevronRight);
-  const [Separator, baseSeparatorProps] = getOverrides(
-    overrides.Separator,
-    StyledSeparator,
+  const [List, baseListProps] = getOverrides(overrides.List, StyledList);
```

Comment:
this way, it would be a breaking change, isn't it? do you think we can get away without one?

Reply:
I'm not sure how we can do that in a way that won't break it. That's why the PR was how it was originally. It shows how you can use the best practice for markup if you want it. You could pass through `Separator` overrides only to the pseudo element but `component` and `props` would be ignored. Or you stick with how it is and override the `Separator` to be `null` and override the `style` on the `ListItem`.

The issue is that currently the separators are injected and they aren't the "best practice" for SEO/accessibility.